plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -927,6 +927,7 @@ private Answer execute(ResizeVolumeCommand cmd) {
                 ClusterMO clusterMO = new ClusterMO(context, morCluster);
 
                 List<Pair<ManagedObjectReference, String>> lstHosts = clusterMO.getClusterHosts();
+                _storageProcessor.rescanAllHosts(context, lstHosts, true, true);
```

Comment:
There's some issue with VMware not able to detect resized/modification in the LUN size, this is a workaround for VMware to see an updated/resized LUN. Otherwise VMFS datastore doesn't see it can resize and silently fails.

Reply:
I hope this won't cause an issue when there are many hosts in one cluster. Not sure if there is a way to limit rescanning to specific hosts or should we limit it to Solidfire volumes?

Reply:
I think we'll need to rescan all the hosts in the cluster; yes that could make the process very slow. If necessary I can add in some logic to scan only when datastore is iscsi (start with -iqn...)

Reply:
@shwstppr I had a code check, line no. 925 ensures this is run only in case of managed storage such as solidfire; so wouldn't affect non-managed storage such as nfs primary storage.

Reply:
I'll still add a check